pkg/ostree/Makefile.am

```diff
@@ -1,5 +1,5 @@
 
-ostreedir = $(pkgdatadir)/ostree
+ostreedir = $(pkgdatadir)/updates
```

Comment:
We can install to /usr/share/cockpit/ostree, and just use "name" in the manifest to rename this package to updates.